bot/exts/evergreen/trivia_quiz.py

```diff
@@ -3,190 +3,211 @@
 import logging
 import operator
 import random
+import re
+import string
+import typing as t
```

Comment:
This and some of the other typing annotations are going to conflict with the current codebase. You should pull down the recent lancebot and resolve merge conflicts.